src/components/permissions/routes.ts

```diff
@@ -91,4 +91,35 @@ const routeUpdateRowPolicies = async (request: any, reply: any) => {
   return reply.send(await updateRowPolicies())
 }
 
-export { routeUserInfo, routeLogin, routeLoginOrg, routeUpdateRowPolicies, routeCreateHash }
+const routeVerification = async (request: any, reply: any) => {
+  const { uid } = request.query
```

Comment:
Hmmm was wondering how we would do this server-side localisation. Have this been considered by @sworup? I have seen some strings un-localised as part of plugins, which should be tricker to solve now, but this ones could possibly be listed in a similar file as in the Front-end.

Reply:
Good point, but I'd say localising server-generated messages would be low priority for now.